http/src/main/resources/init-resources/shutdown.sh

```diff
@@ -3,10 +3,22 @@
 set -e -x
 
 ##
-# This is a shutdown script designed to run on Leo-created Dataproc clusters.
+# This is a shutdown script designed to run on Leo-created Google Dataproc clusters and Google Compute Engines (GCE).
 ##
 
-# Templated values
+# Runtimes created after September 2024 should have the CLOUD_SERVICE env exported, but for older runtimes, the CLOUD_SERVICE
```

Comment:
Exports in `gce-init.sh`/`init-actions.sh` won't carry over to `startup.sh` /`shutdown.sh`. So I think the code is correct to always export it here, maybe remove the part in the comment about `Runtimes created after September 2024 should have the CLOUD_SERVICE env exported`

Reply:
Gotcha, we should embed this in the runtime template then, but this will do for now. at least it is a bit clearer